ONSdigital/design-system
# Fieldset and question can convert legend to page title

### What is the context of this PR?
Added option to make legend a `H1` and a question title.

Usage:
- When using the question macro include the parameter `"legendIsPageTitle": true` inside the `onsQuestion` configuration. Also include `"dontWrap": true` in the configuration of the input you are using.
- When just using an input component without the question component only include the parameter `"legendIsPageTitle": true` in the configuration of the input you are using.

https://trello.com/c/3VplFHOj

### How to review 
- Ensure all fieldset calling components work as expected.
- Check accessibility
